ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/http/services/EventStreamResponseConverter.java

```diff
@@ -19,15 +20,16 @@ public class EventStreamResponseConverter {
 
 	public static final String CUSTOM = "http://example.org/";
 	public static final Property MEMBER_TYPE = createProperty(CUSTOM, "memberType");
+	private final String hostname;
 	private final ViewSpecificationConverter viewSpecificationConverter;
 
-	public EventStreamResponseConverter(ViewSpecificationConverter viewSpecificationConverter) {
+	public EventStreamResponseConverter(AppConfig appConfig, ViewSpecificationConverter viewSpecificationConverter) {
```

Comment:
Since we only need the hostname, and we don't expect to need more from the appConfig in the future (this object may even be removed), it may be better to replace this by `String hostName`. Then this class can be instantiated easier as well, as you only need a String instead of an entire AppConfig instance.